gitconfig.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+gitConfig () {
+    git config --global user.email "$email"
+    git config --global user.name "$username"
+    git config --global push.default simple
+}
+
+read -p "What is your real name? " username
+read -p "What is your Github email address? " email
```

Comment:
@alayek there is absolutely no email validation for the git config command. We might need to add email validation to this script, do you think we should do it for the first release? I am not familiar with email validation yet. Must be some snippets or framework out there for bash script.